@caelinsutch 
```
  <<< failure message
    -- run_test.py: invoking following command in '/root/project/src/image_pipeline/image_proc':
     - /opt/ros/dashing/bin/ament_flake8 --xunit-file /root/project/build/image_proc/test_results/image_proc/flake8.xunit.xml
    
    ./launch/image_proc.launch.py:33:1: F401 'sys' imported but unused
    import sys
    ^
    
    ./launch/image_proc.launch.py:35:1: F401 'launch.LaunchIntrospector' imported but unused
    from launch import LaunchDescription, LaunchIntrospector, LaunchService
    ^
    
    ./launch/image_proc.launch.py:35:1: F401 'launch.LaunchService' imported but unused
    from launch import LaunchDescription, LaunchIntrospector, LaunchService
    ^
    
    ./launch/image_proc.launch.py:36:1: F401 'launch_ros.get_default_launch_description' imported but unused
    from launch_ros import actions, get_default_launch_description
    ^
    
    4     F401 'sys' imported but unused
    
    1 files checked
    4 errors
    
    'F'-type errors: 4
    
    Checked files:
    
    * ./launch/image_proc.launch.py
    -- run_test.py: return code 1
    -- run_test.py: verify result file '/root/project/build/image_proc/test_results/image_proc/flake8.xunit.xml'
  >>>
build/image_proc/test_results/image_proc/flake8.xunit.xml: 4 tests, 0 errors, 4 failures, 0 skipped
- image_proc.flake8 F401 (./launch/image_proc.launch.py:33:1)
  <<< failure message
    'sys' imported but unused:
    import sys
  >>>
- image_proc.flake8 F401 (./launch/image_proc.launch.py:35:1)
  <<< failure message
    'launch.LaunchIntrospector' imported but unused:
    from launch import LaunchDescription, LaunchIntrospector, LaunchService
  >>>
- image_proc.flake8 F401 (./launch/image_proc.launch.py:35:1)
  <<< failure message
    'launch.LaunchService' imported but unused:
    from launch import LaunchDescription, LaunchIntrospector, LaunchService
  >>>
- image_proc.flake8 F401 (./launch/image_proc.launch.py:36:1)
  <<< failure message
    'launch_ros.get_default_launch_description' imported but unused:
    from launch_ros import actions, get_default_launch_description
  >>>
```

Run ament_flake8 locally to find these issues. You can also re-add the remaps to be complete as well, I'd appreciate that.

--- a/image_proc/launch/image_proc.launch.py
+++ b/image_proc/launch/image_proc.launch.py
@@ -30,10 +30,8 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-import sys
-
-from launch import LaunchDescription, LaunchIntrospector, LaunchService
-from launch_ros import actions, get_default_launch_description
+from launch import LaunchDescription
+from launch_ros import actions
 
 from launch_ros.descriptions import ComposableNode
 
@@ -61,6 +59,8 @@ def generate_launch_description():
                 remappings=[
                     # Subscriber remap
                     ('image', 'image_mono'),
+                    ('camera_info', 'camera_info'),
+                    ('image_rect', 'image_rect')
                 ],
             ),
             # Example of rectifying an image

